src/ckeditor/ckConfigurator.js

```diff
@@ -666,6 +667,11 @@ const ckConfigurator = (function () {
         // add the toolbar
         if (typeof toolbars[toolbarType] !== 'undefined') {
             ckConfig.toolbar = toolbars[toolbarType];
+
+            //enable sourcedialog plugin upon featureflag (false by default)
+            if(context.featureFlags.FEATURE_FLAG_CKEDITOR_SOURCEDIALOG) {
```

Comment:
**question:** Are you sure the context will always contain a `featureFlags` entry?

**suggestion:** Add a path check
```suggestion
            if (context.featureFlags && context.featureFlags.FEATURE_FLAG_CKEDITOR_SOURCEDIALOG) {
```

Reply:
Thanks, nice catch! Fixed upon your suggestion